src/components/Content/Content.module.scss

```diff
@@ -0,0 +1,12 @@
+@import "../../scss/bootstrap.scss";
+
+.largePadding {
+  padding: 3em 2em 1em 2em;
+
+  @include media-breakpoint-up(sm) {
+    padding: 4em 3em 2em 3em;
+  }
+  @include media-breakpoint-up(lg) {
+    padding: 6em 0 4em 0;
+  }
+}
```

Comment:
I could not convert all paddings to bootstrap classes, because some are too large.
For consistency I kept all paddings in a single place.

Alternatively: we could define custom spacers for the homepage: https://getbootstrap.com/docs/4.3/utilities/spacing/